packages/amplify_datastore/example/lib/main.dart

```diff
@@ -260,7 +260,11 @@ class _MyAppState extends State<MyApp> {
 
   savePost(String title, int rating, Blog associatedBlog) async {
     try {
-      Post post = Post(title: title, rating: rating, blog: associatedBlog);
+      Post post = Post(
+          title: title,
+          rating: rating,
+          created: TemporalDateTime.now(),
```

Comment:
I'm wondering if it would be better that the `created` timestamp can be generated internally by the `Post` class, as it's more an "internal" field, devs could be less concerned about how to assign it a correct value.

Reply:
Not quite, this is a requirement from nullsafety.  Post has a required created field defined in its schema graphql that must be set, before that wasn't being enforced by the compiler so we missed creating their created field. 

You have a great point though.  There is actually some new Codegen work from Aaron Lee to automatically create "createdAt" and "lastUpdatedAt" fields.  We need to update our code to do this which will be done in the future. 

The created field you see here is more just to check that our internal TemporalDateTime class is working.

Reply:
Thanks for the explanation!
Hence... before this work gets completed:
>We need to update our code to do this which will be done in the future.

Developers would need to insert `created` timestamp when create every new object, correct? And after the work the developers can omit `created`. If that's the case, I think to avoid this inconsistency would be nice. Thoughts?

Reply:
Yes as it would be created automatically.  How do you propose we avoid this inconsistency for now though?  Please remember these are our example app codegen models what fields the users create is up to them.